### PR TITLE
testing/jitterentropy-rngd: new aport

### PR DESCRIPTION
**Pull request by HRio**

### CODE REVIEW

File: testing/jitterentropy-rngd/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,32 @@
+# Contributor: Henrik Riomar <henrik.riomar@gmail.com>
+# Maintainer:
+pkgname=jitterentropy-rngd
+pkgver=1.0.8
+pkgrel=0
+pkgdesc="CPU Jitter RNG Daemon"
+url="http://www.chronox.de/jent.html"
+arch="all"
+license="GPL-2.0 BSD"
```
Comment (maxice8) [inline testing/jitterentropy-rngd/APKBUILD]: BSD isn't spdx and is also ambiguous

GPL-2.0 needs -or-later or -only suffix as per SPDX 3.0

File: testing/jitterentropy-rngd/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+makedepends="linux-headers"
+options="!check" # no test suite
+subpackages="$pkgname-doc $pkgname-openrc"
+source="jitterentropy-rngd-$pkgver.tar.gz::https://github.com/smuellerDD/jitterentropy-rngd/archive/v$pkgver.tar.gz
+	$pkgname.initd
+	"
+builddir="$srcdir/jitterentropy-rngd-$pkgver"
+
+build() {
+	cd "$builddir"
```
Comment (Ikke) [inline testing/jitterentropy-rngd/APKBUILD]: `cd "$builddir"` is obsolete nowadays and can be removed (as abuild does that automatically).

File: testing/jitterentropy-rngd/APKBUILD (hunk 1 above)
Comment (HRio) [reply testing/jitterentropy-rngd/APKBUILD]: thanks, is `"GPL-2.0-only BSD-3-Clause"` OK? when its GPL or BSD (not GPL+BSD)

File: testing/jitterentropy-rngd/APKBUILD (hunk 2 above)
Comment (HRio) [reply testing/jitterentropy-rngd/APKBUILD]: OK, any news on when `newapkbuild` will stop adding it?

File: testing/jitterentropy-rngd/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,32 @@
+# Contributor: Henrik Riomar <henrik.riomar@gmail.com>
+# Maintainer:
```
Comment (Ikke) [inline testing/jitterentropy-rngd/APKBUILD]: You don't want to maintain this?

File: testing/jitterentropy-rngd/APKBUILD (hunk 2 above)
Comment (Ikke) [reply testing/jitterentropy-rngd/APKBUILD]: I submitted a PR to remove it from the template.

File: testing/jitterentropy-rngd/APKBUILD (hunk 3 above)
Comment (HRio) [reply testing/jitterentropy-rngd/APKBUILD]: I am adding it as something to test for people that report issues with`haveged`, somebody mentioned it on irc the other day I think. If this package does not work better then `haveged` in their systems, we can drop `jitterentropy-rngd` package.

File: testing/jitterentropy-rngd/APKBUILD (hunk 2 above)
Comment (HRio) [reply testing/jitterentropy-rngd/APKBUILD]: thanks, will update this PR.

File: testing/jitterentropy-rngd/APKBUILD (hunk 1 above)
Comment (maxice8) [reply testing/jitterentropy-rngd/APKBUILD]: check the source code that are under GPL-2.0- , they will have a 

`this is licensed under GNU GPL 2, **or later**` or `this is licensed under GNU GPL 2 **only**`

Comment (HRio) [reply testing/jitterentropy-rngd/APKBUILD]: there is no later for gpl in the sources.

My question is, whats the proper form for license, when all files are dual licensed. I.e. its not some files bsd and some gpl. But all are dual.

Comment (HRio) [reply testing/jitterentropy-rngd/APKBUILD]: I changed to "GPL-2.0-only BSD-3-Clause"

**Pull request closed without merging**